### PR TITLE
Update content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

### PR DESCRIPTION
**Pull request by jailton**
Update translation of "**Usando Minikube para criar um cluster**" to use the "control plane" terminology, and follow the translation guide to Portuguese.

Ref: https://docs.google.com/spreadsheets/d/1HiaVFJ7nRZo1MPNNYyq1u-L_N9xrJvzuUzkZ3GuJGNU/edit#gid=325775340

Preview: https://deploy-preview-26756--kubernetes-io-master-staging.netlify.app/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/

/language pt
/kind cleanup 
/wg naming

### CODE REVIEW

File: content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 1]
```diff
@@ -75,22 +75,22 @@ <h2 style="color: #3771e3;">Diagrama de Cluster</h2>
...
+                <p>Um cluster Kubernetes pode ser implantado em máquinas físicas ou virtuais. Para começar o desenvolvimento do Kubernetes, você pode usar o Minikube. O Minikube é uma implementação leve do Kubernetes que cria uma VM em sua máquina local e implanta um cluster simples contendo apenas um nó. O Minikube está disponível para sistemas Linux, macOS e Windows. O linha de comando <i>(cli)</i> do Minikube fornece operações básicas de inicialização para trabalhar com seu cluster, incluindo iniciar, parar, status e excluir. Para este tutorial, no entanto, você usará um terminal online fornecido com o Minikube pré-instalado.</p>
```
Comment (rikatz) [inline content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:93]: "A linha de comando do Minikube" :)

After that, lgtm

**Review by rikatz — APPROVED** [review]

File: content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 2]
```diff
@@ -29,16 +29,16 @@ <h3>Objetivos</h3>
...
+                Para fazer uso desse novo modelo de implantação, os aplicativos precisam ser empacotados de uma forma que os desacoplem dos hosts individuais: eles precisam ser empacotados em contêineres. Os aplicativos em contêineres são mais flexíveis e disponíveis do que nos modelos de implantação anteriores, nos quais os aplicativos eram instalados diretamente em máquinas específicas como pacotes profundamente integrados ao host. 
                 <b> O Kubernetes automatiza a distribuição e o agendamento de contêineres de aplicativos em um cluster de maneira mais eficiente. </b> 
                 O Kubernetes é uma plataforma de código aberto e está pronto para produção.
                 </p>
                 <p>Um cluster Kubernetes consiste em dois tipos de recursos:
                     <ul>
-                        <li>O <b>Master</b> coordena o cluster </li>
-                        <li>Os <b>Nodes</b> são os trabalhadores que executam aplicativos</li>
+                        <li>A <b>Camada de gerenciamento <i>(Control Plane)</i></b> coordena o cluster </li>
```
Comment (devlware) [inline content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:40]: IMO we could call "Control Plane" "Plano de controle". Even google is using this term in their [translations](https://cloud.google.com/anthos/clusters/docs/aws/concepts/architecture?hl=pt-br#management_cluster).
Camada de Gerenciamento looks like to me that we have a layer over another or something like that.
But from the K8s [docs](https://d33wubrfki0l68.cloudfront.net/2475489eaf20163ec0f54ddc1d92aa8d4c87c96b/e7c81/images/docs/components-of-kubernetes.svg) its more like a group of softwares that have a purpose.

What do you think @jailton and @rikatz?

**Review by devlware — COMMENTED** [review]
Hi @jailton, thanks for updating the docs.

File: content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html (hunk 2 above)
Comment (rikatz) [reply content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:40]: To be really honest, I've hard feelings about this. Some of Google translations are automated, and I guess some things actually don't make sense to the end users. I've seen some authors translating slices to "fatias" where this should be kept as slices :)

We've discussed that some words make more sense staying in English (for Portuguese translations) and putting them in Italic. I don't think this is the case, BUT "Plano de Controle" to me represents less than "Camada de gerenciamento" :)

On a past translation I did, I was discussing with @jcjesus about the proper translation and we've reached the consensus of going with "Camada de Gerenciamento" due to a lack of a better translation :)

But this is my opinion, would like to listen other folks

Comment (jailton) [reply content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:40]: Hi, @devlware. I'll send a message on our channel to discuss this issue.

**Comment by devlware** [comment]
@jailton You mentioned that you would run a call with more people to discuss directions on the translation.
Should we proceed with the translation or keep the word in English?

**Comment by jailton** [comment]
@devlware We'll discuss the direction on translation in our next meeting. I think it'll be on next Tuesday.

**Comment by rikatz** [comment]
We need to schedule this meeting ...

Anyone up to conducting this next time? I can do next tuesday, 10 AM BRT

**Comment by yagonobre** [comment]
@rkatz works for me

**Review by jcjesus — APPROVED** [review]

**Pull request merged**